plugins/source/mysql/client/transformer.go

```diff
@@ -135,3 +140,41 @@ func (*Transformer) TransformTextArray(v *schema.TextArray) any {
 
 	return v.String()
 }
+
+func (*Transformer) RecordToCQTypes(table *schema.Table, record arrow.Record) (schema.CQTypes, error) {
```

Comment:
Let's just bring back the old `testData` that generates CQTypes directly. this way this migration will be much easier (same as in oracleDB)

Reply:
https://github.com/cloudquery/plugin-sdk/pull/811

Reply:
Done in a8c4fb63be7ed5a1c855e3125466a8cda2be0917